lib/screens/v2/widget_instance/alert.ex

```diff
@@ -136,6 +139,7 @@ defmodule Screens.V2.WidgetInstance.Alert do
     defp serialize_icon(unquote(e)), do: unquote(icon)
   end
 
+  # possibly useful for other alerts
```

Comment:
I could see this potentially being useful for other widgets, too. A very standard `Util` function.